images/custom/Containerfile

```diff
@@ -64,7 +64,7 @@ RUN useradd -ms /bin/bash frappe \
     && chown -R frappe:frappe /var/log/nginx \
     && chown -R frappe:frappe /var/lib/nginx \
     && chown -R frappe:frappe /run/nginx.pid \
-    && chmod +x /usr/local/bin/nginx-entrypoint.sh
+    && chmod 755 /usr/local/bin/nginx-entrypoint.sh
```

Comment:
Add slash at the end to allow for next line "&& chmod 755 /usr/local/bin/nginx-entrypoint.sh \"